src/image.rs

```diff
@@ -1184,6 +1206,87 @@ pub struct ProgressDetail {
     total: Option<u64>,
 }
 
+/// Describes query parameters for DELETE /images/{name}
+/// https://docs.docker.com/engine/api/v1.42/#tag/Image/operation/ImageDelete
+#[derive(Default, Debug)]
+pub struct DeleteOptions {
```

Comment:
To follow the pattern already used by shiplift, I think the new options structs also need builders.